src/test/java/ui/UserAccount.java

```diff
@@ -0,0 +1,43 @@
+package ui;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+
+import javax.annotation.Nullable;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserAccount {
+    @NonNull
+    private String firstName;
+    @NonNull
```

Comment:
if you decide to use `@NonNull` and `@Nulable`, it would be nice to use them in all classes/methods for static analysis.